observationtree/src/main/scala/explore/observationtree/TargetObsList.scala

```diff
@@ -168,65 +212,85 @@ object TargetObsList {
                           .exists(_ === targetId)
                           .fold("right", "down")
-                      )(^.cursor.pointer, ^.onClick --> toggleCollapsed(targetId)),
+                      )(^.cursor.pointer,
+                        ^.onClick ==> { e: ReactEvent =>
+                          e.stopPropagationCB >> toggleCollapsed(targetId)
+                        }
+                      ),
                       Icon("chevron right")
                     )
 
-                  Droppable(target.name) {
-                    case (provided, _ /*snapshot*/ ) =>
+                  Droppable(target.id.toString) {
+                    case (provided, snapshot) =>
                       <.div(
                         provided.innerRef,
-                        provided.droppableProps
-                        // getListStyle(snapshot.isDraggingOver)
+                        provided.droppableProps,
+                        getListStyle(snapshot.isDraggingOver)
                       )(
-                        <.span(
-                          opIcon,
+                        Segment(vertical = true,
+                                raised = props.focused.get
+                                  .exists(_ === FocusedTarget(target.id)),
+                                clazz = GPPStyles.ObsTreeGroup
+                        )(
+                          ^.cursor.pointer,
+                          ^.onClick --> props.focused.set(FocusedTarget(targetId).some).runInCB
+                        )(
                           <.span(
-                            target.name,
-                            <.span(^.float.right, s"$obsCount Obs"),
-                            ^.cursor.pointer,
-                            ^.onClick --> props.focused.set(FocusedTarget(targetId).some).runInCB
-                          )
-                        ),
-                        TagMod.when(!state.collapsedTargetIds.contains(targetId))(
-                          targetObs.zipWithIndex.toTagMod {
-                            case (obs, idx) =>
-                              <.div(GPPStyles.ObsTreeItem)(
-                                Draggable(obs.id.toString, idx) {
-                                  case (provided, snapshot, _) =>
-                                    def dragIcon =
-                                      <.span(
-                                        provided.dragHandleProps,
-                                        Icon("sort")
-                                      )
+                            opIcon,
+                            <.span(
+                              // Segment(raised =
+                              //   props.focused.get
+                              //     .exists(_ === FocusedTarget(target.id))
+                              // )(
+                              target.name,
+                              <.span(^.float.right, s"$obsCount Obs")
```

Comment:
This is low priority. Are we using the float to do layout, aligning to the right? Could we use flexbox instead?

Reply:
Good idea, I'll try that. Also deleting that commented out code.